api/openapi.json

```
@@ -277,6 +277,10 @@
                         "readOnly": true,
                         "type": "string"
                     },
+                    "content_type": {
+                        "description": "Content Type (rpm) of the repository",
```

```suggestion
                        "description": "Content Type of the repository (rpm) ",
```
to be consistent . More content types will be added in future.